### PR TITLE
HCL Wash Changes

### PR DESCRIPTION
**Pull request by knowds**
@mattmurp , please see latest comments in https://jira01.hclpnp.com/browse/ASA-985 for changes made and pending changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 
-Not yet a customer of IBM Application Security on Cloud? Click [here](http://ibm.biz/ASoC-FromIDE) for a free trial of Application Security on Cloud to use with this plugin
+Not yet a customer of HCL AppScan on Cloud? Click [here](https://cloud.appscan.com) for a free trial of Application Security on Cloud to use with this plugin
 
 # Prerequisites:
 
-- An account on the [IBM Application Security on Cloud](https://www.ibm.com/marketplace/cloud/application-security-on-cloud/) service. You'll need to [create an application](http://www.ibm.com/support/knowledgecenter/SSYJJF_1.0.0/ApplicationSecurityonCloud/ent_create_application.html) on the service to associate your scans with.
+- An account on the [HCL AppScan on Cloud](https://cloud.appscan.com/) service. You'll need to [create an application](http://www.ibm.com/support/knowledgecenter/SSYJJF_1.0.0/ApplicationSecurityonCloud/ent_create_application.html) on the service to associate your scans with.
```
Comment (mattmurp) [inline README.md]: Can you update the link for creating an application to:  https://help.hcltechsw.com/appscan/ASoC/ent_create_application.html

File: build.gradle [hunk 2]
```diff
@@ -16,20 +16,20 @@ dependencies {
...
 
-group = 'com.ibm.security'
-version = '1.0.2'
+group = 'com.hcl.security'
+version = '1.0.0'
 
 pluginBundle {
-	website = 'https://www.ibm.com/us-en/marketplace/application-security-on-cloud'
+	website = 'https://www.ibm.com/support/knowledgecenter/en/SSYJJF_1.0.0/ApplicationSecurityonCloud/src_irx_gen_gradle.html'
 	vcsUrl = 'https://github.com/hclproducts/application-security-gradle-plugin'
```
Comment (mattmurp) [inline build.gradle]: Can you update this link to:
https://help.hcltechsw.com/appscan/ASoC/src_irx_gen_gradle.html

File: README.md (hunk 1 above)
Comment (knowds) [reply README.md]: @mattmurp , suggested change has been committed.

File: build.gradle (hunk 2 above)
Comment (knowds) [reply build.gradle]: @mattmurp , suggested change has been committed.

**Pull request merged**